plugins/hls-eval-plugin/src/Ide/Plugin/Eval/Types.hs

```diff
@@ -55,6 +55,8 @@ type Loc = Located Line
 
 type Line = Int
 
+{- HLINT ignore locate "Use zipWithFrom" -}
+
```

Comment:
If we do not use the rule for code like this, then will we ever use this rule?
If we won't, I think it's better to put this in the hlint setting.

Reply:
You're right. I'll just apply this hint too.